### PR TITLE
Grouping FrameworkBundle upgrade notes

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| License       | MIT

### CODE REVIEW

**Comment by chalasr** [comment]
👍 

Status: reviewed

**Comment by javiereguiluz** [comment]
👍

File: UPGRADE-3.2.md [hunk 1]
```diff
@@ -69,12 +71,6 @@ Form
    }
    ```
 
-FrameworkBundle
----------------
-
- * The service `serializer.mapping.cache.doctrine.apc` is deprecated. APCu should now
-   be automatically used when available.
```
Comment (xabbuh) [inline UPGRADE-3.2.md]: What about moving the other items here? FrameworkBundle would then be sorted properly into the sections.

Comment (yceruto) [inline UPGRADE-3.2.md]: Indeed, sorting 4.0 also

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @yceruto.

**Pull request closed without merging**